### PR TITLE
[14.0][MIG]sale_line_refund_to_invoice_qty: Migration to 14.0

### PR DESCRIPTION
**Pull request by GuillemCForgeFlow**
@ForgeFlow

### CODE REVIEW

File: sale_line_refund_to_invoice_qty/migrations/13.0.1.1.1/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 ForgeFlow <http://www.forgeflow.com>
```
Comment (ChrisOForgeFlow) [inline sale_line_refund_to_invoice_qty/migrations/13.0.1.1.1/post-migration.py]: Please remove this file

File: sale_line_refund_to_invoice_qty/migrations/13.0.1.1.1/pre-migration.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Copyright 2021 ForgeFlow <http://www.forgeflow.com>
```
Comment (ChrisOForgeFlow) [inline sale_line_refund_to_invoice_qty/migrations/13.0.1.1.1/pre-migration.py]: Please remove this file

**Review by ChrisOForgeFlow — CHANGES_REQUESTED** [review]
Please remove migrations scripts for v13

**Comment by GuillemCForgeFlow** [comment]
> Please remove migrations scripts for v13


Done! Thanks for the review :smiley:

**Review by ChrisOForgeFlow — APPROVED** [review]
+1

**Review by JoanMForgeFlow — APPROVED** [review]
+1

**Review by AaronHForgeFlow — APPROVED** [review]
Code review LGTM

cc @JordiMForgeFlow

**Comment by bosd** [comment]
Ping @oca-maintainers

**Pull request merged**